src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -143,4 +137,4 @@ boolean verifyHostName(String toVerify) {
     return matcher.matches();
   }
 
-}
\ No newline at end of file
+}
```

Comment:
I'm curious as to what you used to reformat the file? Did you accomplish this manually, or did you use a tool? 

If you created these formatting changes manually, I would recommend using the [google-java-format](https://github.com/google/google-java-format) plugin to automatically reformat your file according to the Google Java Style Guide. (That's what I'm currently using!) 

If you're interested in adding this plugin to IntelliJ, you can follow these steps: 
1. Go to `IntelliJ IDEA` > `Preferences` > `Plugins`
2. Search for `google-java-format` and install the plugin
3. Once installed, you can hit `command`+`option`+`L` to reformat the file (if using a Mac) 

You'll see that this will remove that extra white space on line 139 and make other minor formatting changes for you.